projects/rocprofiler-sdk/source/lib/python/rocpd/source/perfetto.cpp

```diff
@@ -300,7 +309,7 @@ write_perfetto(
             else
                 _namess << "(UNK)";
 
-            auto _track = ::perfetto::Track{get_hash_id(_namess.str()), this_pid_track};
+            auto _track = ::perfetto::Track::Global(++global_track_index);
```

Comment:
```suggestion
            auto _track = ::perfetto::Track{get_hash_id(_namess.str()), this_pid_track};
```
Don't really like the global_track_index.  Can we go back to the original?

Reply:
Done.